internal/controllers/parser.go

```diff
@@ -310,6 +317,23 @@ func UpdateConfigFromAPIOpts(
 
 				clusterName := generateClusterName(upstreamServiceHost, upstreamServicePort)
 
+				var authHeaders []*envoy_config_core_v3.HeaderValue
+
+				if finalOpts.Auth.Scheme == "cloudentity" {
+					clBuilder.AddAPI(upstreamServiceHost, upstreamServicePort, name, name, routePath, method)
+					authHeaders = []*envoy_config_core_v3.HeaderValue{
+						&envoy_config_core_v3.HeaderValue{
```

Comment:
Nitpicking here: no need to pass the type here `envoy_config_core_v3.HeaderValue` as it is already known

Reply:
I think I need to, as I'm using `&`, this doesn't compile:
```
					authHeaders = []*envoy_config_core_v3.HeaderValue{
						&{
							Key:   cloudentity.HeaderAuthorizerURL,
							Value: fmt.Sprintf("https://%s:%d", upstreamServiceHost, upstreamServicePort),
						},

```

Reply:
A linter is warning about it. But I don't think it really matters. 
```
[{
	"resource": "/root/go/src/github.com/kubeshop/kusk-gateway/internal/controllers/parser.go",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 4,
	"message": "redundant type from array, slice, or map composite literal",
	"source": "simplifycompositelit",
	"startLineNumber": 325,
	"startColumn": 7,
	"endLineNumber": 325,
	"endColumn": 40,
	"tags": [
		1
	]
}]
```

Reply:
Found the issue, updated it, thanks!